BUILD.md

```diff
@@ -1,66 +1,138 @@
-# WAF Build #
+# Building ArduPilot #
 
 Ardupilot is gradually moving from the make-based build system to
-[Waf](https://waf.io/).
+[Waf](https://waf.io/). You can read the [Waf Book](https://waf.io/book/) if
+you want to learn more about Waf.
 
-To keep access to Waf convenient, use the following alias from the
-root ardupilot directory:
+Waf should always be called from the ardupilot's root directory. Differently
+from the make-based build, with Waf there's a configure step to choose the
+board to be used (default is `sitl`).
 
-```bash
-alias waf="$PWD/modules/waf/waf-light"
-```
+## Basic usage ##
 
-You can also define the alias or create a function in your shell rc file (e.g.
-`~/.bashrc`).
+There are several commands in the build system for advanced usages, but here we
+list some basic and more used commands as example.
 
-You can read the [Waf Book](https://waf.io/book/) if you want to learn more
-about Waf.
+* **Build ArduCopter**
 
-## Calling waf ##
+    Here we use minlure as an example of Linux board. See more advanced usages
```

Comment:
_See more advanced usages_  but the list of supported boards is in the basic section